deepchecks/checks/performance/regression_error_distribution.py

```diff
@@ -0,0 +1,113 @@
+# ----------------------------------------------------------------------------
+#
+"""The regression_error_distribution check module."""
+import matplotlib.pyplot as plt
+import pandas as pd
+from scipy.stats import kurtosis, kurtosistest
+from sklearn.base import BaseEstimator
+
+from deepchecks import CheckResult, Dataset, SingleDatasetBaseCheck, ConditionResult
+from deepchecks.utils.metrics import ModelType, task_type_validation
+from deepchecks.utils.strings import format_number
+
+__all__ = ['RegressionErrorDistribution']
+
+
+class RegressionErrorDistribution(SingleDatasetBaseCheck):
+    """Check regresstion error distribution.
+
+    Args:
+        n_top_samples (int): amount of samples to show which are of Largest under / over estimation errors.
+        alternative (str): Defines the alternative hypothesis to calculate the p value
+            ‘two-sided’: the kurtosis of the distribution underlying the sample is different from that of
+                         the normal distribution
+            ‘less’: the kurtosis of the distribution underlying the sample is less than that of
+                    the normal distribution
+            ‘greater’: the kurtosis of the distribution underlying the sample is greater than that of
+                        the normal distribution
+            (taken from: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.kurtosistest.html)
+            (Only in python 3.7>=)
+    """
+
+    def __init__(self, n_top_samples: int = 3, alternative: str = 'two-sided'):
+        super().__init__()
+        self.n_top_samples = n_top_samples
+        self.alternative = alternative
+
+    def run(self, dataset: Dataset, model: BaseEstimator) -> CheckResult:
+        """Run check.
+
+        Arguments:
+            dataset (Dataset): A dataset object.
+            model (BaseEstimator): A scikit-learn-compatible fitted estimator instance
+
+        Returns:
+           CheckResult:
+                - value is the kurtosis value (Fisher’s definition (normal ==> 0.0)) and the p score of it.
+                - display is histogram of error distirbution and the largest prediction errors.
+
+        Raises:
+            DeepchecksValueError: If the object is not a Dataset instance with a label
+        """
+        return self._regression_error_distribution(dataset, model)
+
+    def _regression_error_distribution(self, dataset: Dataset, model: BaseEstimator):
+        check_name = self.__class__.__name__
+        Dataset.validate_dataset(dataset, check_name)
+        dataset.validate_label(check_name)
+        task_type_validation(model, dataset, [ModelType.REGRESSION], check_name)
+
+        y_test = dataset.label_col
+        y_pred = model.predict(dataset.features_columns)
+
+        diff = y_test - y_pred
+        kurtosis_value = kurtosis(diff)
+        # alternative doesn't work on pytho3.6
+        try:
+            kurtosis_pvalue = kurtosistest(diff, alternative = self.alternative).pvalue
```

Comment:
Why do the switch to p-value? Because it does not depend on the target units?
One problem I can think about concerning p-values is that they do not distinguish between the "fat tail" case and the "narrow distributing" case. We would like to have the default condition fail only on the fat tail case.

Reply:
I went with the logics that if scipy thinks you should diffultivly check on both caes we should too

Reply:
Scipy is checking if the distribution is normal, we're checking if it has fat tails, so it makes sense we'll want different things.
I have no problem with having both the value and p value, and consider the p value only if the value is negative (or whatever side means fat tails)

Reply:
it's problematic because we can't use it in python 3.6

Reply:
I think that because both cases should probably be looked into it's ok, even though maybe one is more important